horace_core/sqw/@rundatah/calc_urange.m

```diff
@@ -1,9 +1,9 @@
-function [urange,u_to_rlu]=calc_urange(obj,varargin)
+function [img_range,u_to_rlu]=calc_urange(obj,varargin)
 % Method calculates q-dE range, this rundata object has
 %
 %Usage:
-%>>[urange,u_to_rlu]=obj.calc_urange()  Calculate urange for fully defined
-%                                      rundatah object
+%>>[urange,u_to_rlu]=obj.calc_urange()  Calculate crystal cartesian range 
```

Comment:
the return value is `img_range` not `urange`

Reply:
ibid